test/examples/express/snapshot.js

```diff
@@ -0,0 +1,455 @@
+// a console.log from a previous run
+module.exports = [
+  {
+    traceId: '2JXQNfGhIadZSxTgeG3joQ==',
```

Comment:
This doesn't look right to me, the snapshot should only contain the data we assert and have control over. The easiest way to fix if we want to stick with this snapshot pattern would be to map OTel's Collector output to entries with just the attributes that are constant, and then `expect().toMatchSnapshot();`

Reply:
Second thing, overall I think that the trace is too long, if a test breaks there's quite a few spans to go through. Can we make it smaller and still test the behaviour we want to test?

Reply:
That's exactly what I'm doing in `assertSpans` - ignoring the parts that are constant, asserting everything else.
I do not want to clean the snapshots up from the non-constant parts because

1. that's more code with next to no benefit.
2. that makes recreating those snapshots more complex and error-prone.
3. looses data(in the context of `traceId` it really doesn't matter, but in some cases it might).

Reply:
I talked to Rauno over Slack, we're going forward with what we have right now.